src/sysinfo/nvml.cpp

```diff
@@ -0,0 +1,83 @@
+#include "nvml.h"
+#include "../util/util_string.h"
+#include "../util/util_log.h"
+
+namespace dxvk {
+    Nvml::Nvml() {
+        const auto nvmlModuleName = "nvml.dll";
+        m_nvmlModule = ::LoadLibraryA(nvmlModuleName);
+        if (m_nvmlModule == nullptr) {
+            if (::GetLastError() != ERROR_MOD_NOT_FOUND) // Ignore library not found
```

Comment:
`auto error = ::GetLastError();` to avoid calling it twice perhaps?

Reply:
Nice find, addressed, thanks.